cardigan/stories/iiif-content/iiif-manifest.js

```diff
@@ -0,0 +1,7535 @@
+const manifest = {
```

Comment:
Maybe this could fo in a mocks directory to use elsewhere?
We need to start having more for testing at some stage I assume.